src/Input/Silk.NET.Input.Common/Interfaces/IMouse.cs

```diff
@@ -66,12 +67,12 @@ public interface IMouse : IInputDevice
         /// <summary>
         /// Called when a single click is performed.
         /// </summary>
-        event Action<IMouse, MouseButton> Click;
+        event Action<IMouse, MouseButton, PointF> Click;
 
         /// <summary>
         /// Called when a double click is performed.
         /// </summary>
-        event Action<IMouse, MouseButton> DoubleClick;
+        event Action<IMouse, MouseButton, PointF> DoubleClick;
```

Comment:
`PointF`?

Reply:
Some windowing backends do have support for subpixel mouse positions, which I presume is why `IMouse.Position` is also PointF.

Reply:
Vector2?

Reply:
Will do all the vector refactorings at the same time as I want to convert Point to Vector2\<int\> and Size to Vector2\<int\> as well.